tox.ini

```diff
@@ -34,7 +34,7 @@ description = Run code formatters and linters.
 [testenv:flake8]
 skip_install = true
 commands =
-    flake8 sssom/ tests/ setup.py
```

Comment:
?

Reply:
`tox` throws an error. I have linked the ticket/discussion where users have complained about it in the PR description